docs/paper/dev/api/folia-support.mdx

```diff
@@ -39,15 +39,15 @@ that can be used for different things. They are:
 If you use these schedulers when running Paper, they will be internally handled to provide the same functionality as if you were
 running Folia.
 
-### Global Scheduler
+### <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/GlobalScheduler.html"}>Global Scheduler</VersionedJavaDocLink>
```

Comment:
Making titles links feels wrong

Reply:
On the line below the Scheduler name also exists so placing it there would work